client/src/pages/Details.js

```diff
@@ -34,6 +35,20 @@ function Details() {
     }
   };
 
+  //const increase
+
+  const increaseCounter = () => {
+    setCounter((count) => count + 1);
+  };
+
+  const decreaseCounter = () => {
+    setCounter((count) => count - 1);
+  };
+
+  const reset = () => {
+    setCounter();
```

Comment:
Although this probably works just fine, it would make your intent clearer if you reset to `0` instead of `undefined`.

```suggestion
    setCounter(0);
```